lib/mix/tasks/ecto.rollback.ex

```diff
@@ -117,6 +124,9 @@ defmodule Mix.Tasks.Ecto.Rollback do
         do: Keyword.merge(opts, log: false, log_migrations_sql: false, log_migrator_sql: false),
         else: opts
 
+    log_level = String.to_existing_atom(Keyword.get(opts, :log_level, "debug"))
+    Logger.configure(level: log_level)
```

Comment:
```suggestion
    if log_level = opts[:log_level] do
      Logger.configure(level: String.to_existing_atom(log_level))
    end
```

Reply:
Yuck forgot to copy paste from `ecto.migrate` to `ecto.rollback`. Apologiesies